src/compas_timber/model/model.py

```diff
@@ -45,18 +48,26 @@ def __from_data__(cls, data):
     def __init__(self, *args, **kwargs):
         super(TimberModel, self).__init__()
         self._beams = []
+        self._plates = []
         self._walls = []
         self._joints = []
         self._topologies = []  # added to avoid calculating multiple times
 
     def __str__(self):
-        return "TimberModel ({}) with {} beam(s) and {} joint(s).".format(self.guid, len(self.beams), len(self.joints))
+        return "TimberModel ({}) with {} beam(s), {} plate(s) and {} joint(s).".format(
```

Comment:
not necessarily related to this PR, but this is getting long. maybe we can format this similarly to `Brep`?
https://github.com/compas-dev/compas/blob/6e85276ce10d485e6b15f895b127c72a6862c7b4/src/compas/geometry/brep/brep.py#L153